roles/foreman_installer/tasks/pre_install.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: 'Ensure known_hosts file exists'
```

Comment:
I think https://docs.ansible.com/ansible/latest/file_module.html would be better to use.

Reply:
From those docs:

>>If file, the file will NOT be created if it does not exist, see the copy or template module if you want that behavior.

Reply:
The known_hosts module doesn't make any statement about creating the file but if it does, then you can use the file resource after setting the known hosts. That prevents wiping and resetting the keys on every run.

Reply:
In the copy module, I need to add `force: no`, and it won't touch the file if it's already there.